.gitmodules

```diff
@@ -1,21 +1,21 @@
-[submodule "oms-events"]
-  path = oms-events
-  url = https://github.com/AEGEE/oms-events.git
 [submodule "oms-global/docker/portal/files"]
   path = oms-global/docker/portal/files
```

Comment:
Can you update these as well? Not sure if global is the proper word, do you have any other words that might replace this?

Reply:
I have no idea actually if we need it, @linuxbandit can you clarify?

Reply:
anyway I'd rather rename/remove it in different PR, as this is not the part of 'shortening the API prefixes' epic, I didn't touch it there

Reply:
indeed, postpone